packages/reassure-cli/src/commands/main.ts

```diff
@@ -0,0 +1,115 @@
+import type { CommandModule } from 'yargs';
+
+import { bye, hello } from '../utils/logger';
+import { run as init } from './init';
+import { run as checkStability } from './check-stability';
+import { MeasureOptions, run as measure } from './measure';
+
+const { Select, Prompt, Confirm } = require('enquirer');
```

Comment:
In RN CLI we use `prompts`, which has TS support once @types/prompts are added. However both libraries have similar footprint, so it's not critical to change

Reply:
that's very fair, I'm happy to refactor in a separate PR.